tests/scheduler/test_modelobs.py

```diff
@@ -43,8 +43,8 @@ def test_ideal(self):
 
         mjd_down = mo_default.downtimes["start"][0] + 0.01
 
-        assert ~mo_default.check_up(mjd_down)
-        assert mo_ideal.check_up(mjd_down)
+        assert ~mo_default.check_up(mjd_down)[0]
+        assert mo_ideal.check_up(mjd_down)[0]
```

Comment:
It would be great to add a test with a short downtime and see that the time returned by check_mjd is the appropriate time, and that if you land in the middle of a day, that it advances to the correct time of sunset.